templates/boot/generic-qemu-boot-template.jinja2

```diff
@@ -23,13 +20,6 @@
 {{ super() }}
 {% endblock %}
 {% block actions %}
-context:
```

Comment:
Please make a separate PR to remove this template block and fix all the existing QEMU entries that rely on these parameters (UEFI).